doc/files/opensc.conf.5.xml.in

```diff
@@ -0,0 +1,1630 @@
+							</para></listitem>
+							<listitem><para>
+									<literal>set_pin_in_specific_context</literal>:
+									<literal>C_SetPIN</literal> in the
+									<literal>CKU_SPECIFIC_CONTEXT</literal>
+									logged session: PUK is passed as the
+									<literal>OldPin</literal> argument of the
+									<literal>C_SetPIN</literal> call.
+							</para></listitem>
+							<listitem><para>
+									<literal>init_pin_in_so_session</literal>:
+									<literal>C_InitPIN</literal> in
+									<literal>CKU_SO</literal> logged session:
+									User PIN 'UNBLOCK' is protected by SOPIN.
+									(PUK == SOPIN).
+							</para></listitem>
+						</itemizedlist>
+					</para></listitem>
+				</varlistentry>
+				<varlistentry>
+					<term>
+						<option>create_puk_slot = <replaceable>bool</replaceable>;</option>
+					</term>
+					<listitem><para>
+							Create slot for unblocking PIN with PUK (Default:
+							<literal>false</literal>). This way PKCS#11 API can
+							be used to login with PUK and change a PIN.  May
+							cause problems with some applications like
+							<application>Firefox</application> and
+							<application>Thunderbird</application>.
+					</para></listitem>
+				</varlistentry>
+				<varlistentry>
+					<term>
+						<option>create_slots_for_pins =  <arg choice="plain"
+								rep="repeat"><replaceable>mode</replaceable></arg>;</option>
+					</term>
+					<listitem><para>
+							Symbolic names of PINs for which slots are created
+							where <replaceable>mode</replaceable> is a list of:
+							<itemizedlist>
+								<listitem><para>
+										<literal>all</literal> (Default): All
+										non-SO-PIN, non-unblocking PINs
+								</para></listitem>
+								<listitem><para>
+										<literal>user</literal>: The first
+										global or first local PIN
+								</para></listitem>
+								<listitem><para>
+										<literal>sign</literal>: The second PIN
+										(first local, second global or second
+										local)
+								</para></listitem>
+							</itemizedlist>
+						</para>
+						<para>
+							Card can contain more then one PINs or more then
+							one on-card application with its own PINs.
+							Normally, to access all of them with the PKCS#11
+							API a slot has to be created for all of them. Many
+							slots could be ennoying for some of widely used
+							application, like FireFox. This configuration
+							parameter allows to select the PIN(s) for which
+							PKCS#11 slot will be created.
+						</para>
+						<para>
+							Only PINs initialised, non-SO-PIN, non-unblocking
+							are associated with symbolic name.
+						</para>
+						<para>
+							For the module to simulate the opensc-onepin module
+							behavior the following option
+							<option>create_slots_for_pins = "user";</option>
+					</para></listitem>
+				</varlistentry>
+			</variablelist>
+		</refsect2>
+
+	</refsect1>
+
+	<refsect1>
+		<title>Environment</title>
```

Comment:
For completeness, I would also list the `OPENSC_DRIVER` and `OPENSC_DEBUG` env variables.